x-pack/test/functional/services/ml/data_visualizer_index_based.ts

```diff
@@ -51,6 +53,167 @@ export function MachineLearningDataVisualizerIndexBasedProvider({
       await testSubjects.existOrFail(`mlFieldDataCard ${fieldName} ${cardType}`);
     },
 
+    async assertNonMetricCardContents(cardType: string, fieldName: string, exampleCount?: number) {
+      await testSubjects.existOrFail(
```

Comment:
Since this assertion for `mlFieldDataCardContent` is used in multiple places I think it would be good to have its own function with `fieldName` and `cardType` as the arguments.

Reply:
Done in https://github.com/elastic/kibana/pull/83174/commits/563f2a3749888bd3ea819800822f913d34b8291f